src/validator/CreditCardVendor.java

```diff
@@ -0,0 +1,32 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardVendor {
+	private String name;
+	private ArrayList<Integer> numberOfChars;
+	private ArrayList<String> IIN;
+	
+	public String getName(){
+		return name;
+	}
+	
+	public ArrayList<Integer> getNumberOfChars(){
+		return numberOfChars;
+	}
+	
+	public ArrayList<String> getIIN(){
+		return IIN;
+	}
+	
+	public CreditCardVendor(String vendorName, ArrayList<Integer> charsNumbers, ArrayList<String> masks){
+		name = vendorName;
+		numberOfChars = charsNumbers;
+		IIN = masks;
+	}
+	
+	@Override
+	public String toString(){
```

Comment:
1. If you need conversion to String in production code - consider ToStringBuilders (e.g. Apache commons)
2. If you stay with "weak" domain - use Immutables.org/Lombok libraries to define only interface and rest (toString etc.) will be generated for you.
3. Define what you really need in toString